.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "suegithub"]
+	path = travis/ZNonPublicDeps
+	url = https://github.com/smootoo/ZNonPublicDeps.git
```

Comment:
We are not allowed to distribute them. e.g. for Oracle http://stackoverflow.com/questions/1074869/find-oracle-jdbc-driver-in-maven-repository/1074971#1074971

The decryption and password are in this script https://github.com/smootoo/slick/blob/travis/travis/extractNonPublicDeps It really just stops people trivially linking to the jars. They aren't hard to get hold of and the links for how to download them are in that script.

Happy to move this repo over to slick org.